### PR TITLE
Consumer implemented in FenrirAPI

### PR DESCRIPTION
**Pull request by Netmisa**
# Introduction
*Consumer implemented in FenrirAPI*

screenshot from 2016-05-18 18 25 48

## 1. Installation
*Execute command:*
```shell
php composer.phar install
php bin/console doctrine:schema:update --force
```

## 2. Run functional/unit tests
*Run consumer from FernirAPI: tyr_event is the queue*
```shell
bin/console rabbitmq:consumer tyr_event
```

*Publish this message in rabbitMQ to tyr_event queue*
```json
{
	"event": "create_user",
	"data": {
		"username": "foo",
		"email": "foo@free.fr",
		"origin": {
			"name": "my_origin"
		}
	}
}
```
```json
{
	"event": "update_user",
	"data": {
		"last_username": "foo",
		"username": "foo42",
		"email": "foo@free.fr",
		"origin": {
			"name": "my_origin"
		}
	}
}
```
```json
{
	"event": "delete_user",
	"data": {
		"username": "foo42"
	}
}
```

Work with @alcalyn

### CODE REVIEW

**Comment by fdegomme-canaltp** [comment]
Vous mettez ce genre de schémas / doc dans Confluence ?

**Comment by Netmisa** [comment]
@fdegomme-canaltp À voir avec l'équipe mais je pense que l'on ferait mieux de mettre la documentation de Fenrir sur github directement, "Ressources/doc" par exemple.

**Comment by vincentlepot** [comment]
IMHO, the command name "rabbitmq:consumer" is too generic. This name should reflect the feature implemented (eg: fenrir:synchronize_users)

**Comment by alcalyn** [comment]
`rabbitmq:consumer` is a command from the Rabbit mq bundle, we have to pass the `tyr_event` argument in order to "listen" to Tyr events (create user, edit...)

File: app/config/config.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ imports:
     - { resource: security.yml }
     - { resource: services.yml }
     - { resource: "@ApiBundle/Resources/config/services.yml" }
+    - { resource: "@TyrSynchroBundle/Resources/config/services.yml" }
```
Comment (vincentlepot) [inline app/config/config.yml]: Is it a

Comment (alcalyn) [inline app/config/config.yml]: Obviously, the sensio bundle generator do it, and there is no DependencyInjection folder in bundles directories

File: app/config/config.yml [hunk 2]
```diff
@@ -81,3 +82,26 @@ fos_rest:
...
+            vhost:    '%rabbit_mq_vhost%'
+            lazy:     false
+            connection_timeout: 3
+            read_write_timeout: 3
+            keepalive: false
+            heartbeat: 0
+    consumers:
+        tyr_event:
+            connection: default
+            exchange_options: {name: 'tyr_event', type: direct}
```
Comment (lrocheWB) [inline app/config/config.yml]: As see together, I have already choose this name "tyr_event_exchange":
https://github.com/lrocheWB/navitia/blob/f6fce066b4e2e60e24f2bf3843eb5f7f4e7d346f/source/tyr/tyr/tyr_user_event.py#L66

**Pull request merged**